SbtProject/build.sbt

```diff
@@ -1,21 +1,20 @@
 name := "ScalaOlio"
 
-version := "1.0"
-
-scalaVersion := "2.11.7"
-
-libraryDependencies += "org.scala-lang" % "scala-reflect"       % "2.11.7"
-
-libraryDependencies += "org.scalatest"  % "scalatest_2.11"      % "2.1.7" % "test"
-
-libraryDependencies += "joda-time"      % "joda-time"           % "2.3"
-
-libraryDependencies += "org.joda"       % "joda-convert"        % "1.5"
-
-libraryDependencies += "org.clapper"    % "grizzled-slf4j_2.11" % "1.0.2"
-
-libraryDependencies += "org.slf4j"      % "slf4j-simple"        % "1.7.7"
-
-libraryDependencies += "org.json4s"     % "json4s-native_2.11"  % "3.2.11"
-
-libraryDependencies += "com.squants"    % "squants_2.11"        % "0.5.3"
+version := "1.0.0"
```

Comment:
Bumped from "1.0" to "1.0.0". Could do that, but really just aligning with semver for now. And this is a _change_, so dependent libs can track it.

That said, I don't feel strongly about _not_ going to a point version. It's just that this isn't my main concentration right now.